.github/workflows/tests.yml

```diff
@@ -190,6 +190,14 @@ jobs:
         # Check service restarted
         if [ $ts3 -lt $ts2 ]; then echo "config check failed: TS2: $ts2 TS3: $ts3"; exit 1; fi
 
+    - name: Test pool operations
+      run: |
+        set +e
+        sudo microceph.ceph osd pool create mypool
+        sudo microceph pool set-rf --size 1
+        sudo microceph pool set-rf --size 3 mypool
+        sudo microceph pool set-rf --size 1 "*"
```

Comment:
Suggest to add something like this for checking
```
sudo microceph pool set-rf --size 1
sudo microceph.ceph config get osd.1 osd_pool_default_size | fgrep -x "1"
sudo microceph pool set-rf --size 3 mypool
sudo microceph.ceph osd pool get mypool size | fgrep -x "size: 3"
sudo microceph pool set-rf --size 1 "*"
sudo microceph.ceph osd pool get mypool size | fgrep -x "size: 1"
```

Reply:
sure thing.